client/my-sites/earn/main.tsx

```diff
@@ -91,9 +98,9 @@ const EarningsMain = ( { section, adsProgramName, query, path } ) => {
 					</AdsWrapper>
 				);
 			case 'payments':
-				return <MembershipsSection section={ section } query={ query } />;
+				return <MembershipsSection query={ query } />;
 			case 'payments-plans':
-				return <MembershipsProductsSection section={ section } />;
+				return <MembershipsProductsSection />;
 
```

Comment:
Section prop is not used in the MembershipsSection or MembershipsProductsSection components, so removed here.